builtin/rebase.c

```diff
@@ -182,13 +184,23 @@ static int run_specific_rebase(struct rebase_options *opts)
 
 int cmd_rebase(int argc, const char **argv, const char *prefix)
 {
+	enum {
+		NO_ACTION,
+		TODO_CONTINUE,
+		TODO_SKIP,
+		TODO_ABORT,
+		TODO_QUIT,
+		TODO_EDIT_TODO,
+		TODO_SHOW_CURRENT_PATCH
+	} action = NO_ACTION;
```

Comment:
I know this was my suggestion, but now that I read it again, it looks odd that the `NO_ACTION` constant does not start with `TODO_`. Maybe prefix it, too?

But then, I think `TODO_*` was a brain fart of mine, anyway, and it should be `ACTION_` anyway.